packages/kit-bg/src/vaults/impls/aptos/Vault.ts

```
@@ -183,6 +202,192 @@ export default class VaultAptos extends VaultBase {
+
+    let actionType = EDecodedTxActionType.UNKNOWN;
+    const payload = rawTx.payload;
+
+    const { sender } = rawTx;
+    const senderAddress = sender.bcsToHex().toString();
+    const actions: IDecodedTxAction[] = [];
+
+    switch (true) {
+      case payload instanceof TransactionPayloadEntryFunction:
+        // eslint-disable-next-line no-case-declarations
+        const functionName = payload.entryFunction.function_name.identifier;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)